en/publish/publish-oppo-instant-games.md

```diff
@@ -83,7 +83,7 @@ Where **game package name**, **game name**,**desktop icon**,**game version name*
 - **Build a release package**
 
   **Build a release package** is optional. The purpose of this check is to build an rpk package that can be published directly. However, there are three prerequisites: **local npm installation path**, **local quickgame-toolkit path**, and **add release signature**. <br>
-  If you do not check the **Build Release Package**, the `rpk` package for testing is built.
+  If you do npsql -U postgres -Wot check the **Build Release Package**, the `rpk` package for testing is built.
```

Comment:
What is `npsql -U postgres -Wot`?

Reply:
let us call it a cut and paste error. fixed. I thought I caught that when I did it.